### PR TITLE
Add Avalara docs

### PR DESCRIPTION
**Pull request by zenweasel**

### CODE REVIEW

File: developer/packages/taxes/avalara.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+The Password provided to you (or created by you) for Avalara. This is the same password
+you would use to log into the administrator dashboard (this is a required field)
+
+#### Company Code
+
+The company code you created in the Avalara administrator panel. (this is a required field)
+
+#### Shipping Tax Code
+
+This is the tax code you want to record Shipping under. By default in Reaction, shipping is not Taxable.
```
Comment (impactmass) [inline developer/packages/taxes/avalara.md]: I think we should add a line about the fact that even though Avalara doesn't list the Shipping Tax Code as required, some functionalities won't work. As such it's labelled as required in the Reaction tax settings.

File: developer/packages/taxes/avalara.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+(but not Tax calculation)
+
+### Selecting Tax Codes
+
+Once the configuration is complete you should see that the Product Detail Page (PDP) has a "Tax Code" field
+that has been pre-populated with Tax Codes from Avalara. You can also enter any tax code here. This is the tax code
+that will be used when the transaction is sent to Avalara.
+
+![avalara-tax-code-dropdown](/assets/avalara-tax-code-dropdown.png)
+
```
Comment (impactmass) [inline developer/packages/taxes/avalara.md]: #1. A line about the "Test Credentials" button is needed here. 
#2. Also a note about setting "Tax Exempt" for accounts via the Account Management screen
Should I write it and push to your branch?

File: admin/taxes.md [hunk 3]
```diff
@@ -26,10 +26,13 @@ If you use Avalara or TaxCloud to find a matching rate, that rate will override
 
 ![](/assets/admin-dashboard-taxes-avalara.png "Avalara Configuration")
```
Comment (impactmass) [inline admin/taxes.md]: This is minor, but I think this screenshot is from an older version. Should we change it to a newer version?

File: developer/packages/taxes/avalara.md (hunk 2 above)
Comment (zenweasel) [inline developer/packages/taxes/avalara.md]: Oh yes, you are correct. If you could add that, that would be great.

Comment (impactmass) [inline developer/packages/taxes/avalara.md]: Added.

File: admin/taxes.md (hunk 3 above)
Comment (zenweasel) [inline admin/taxes.md]: I started to but decided we should wait until all those admin shots get updated.

File: developer/packages/taxes/avalara.md (hunk 1 above)
Comment (zenweasel) [inline developer/packages/taxes/avalara.md]: Yes, fair point, I will add

Comment (zenweasel) [inline developer/packages/taxes/avalara.md]: Done

**Pull request merged**